eng/config/BannedSymbols.txt

```diff
@@ -69,4 +69,5 @@ M:Microsoft.CodeAnalysis.CodeActions.CodeAction.GetOperationsAsync(System.Thread
 M:Microsoft.CodeAnalysis.CodeActions.CodeAction.GetPreviewOperationsAsync(System.Threading.CancellationToken); Use overload that takes a solution
 M:Microsoft.CodeAnalysis.CodeActions.CodeAction.ComputeOperationsAsync(System.Threading.CancellationToken); Use overload that takes progress
 M:Microsoft.CodeAnalysis.CodeActions.CodeAction.GetChangedSolutionAsync(CSystem.Threading.CancellationToken); Use overload that takes progress
```

Comment:
```suggestion
M:Microsoft.CodeAnalysis.CodeActions.CodeAction.GetChangedSolutionAsync(System.Threading.CancellationToken); Use overload that takes progress
```

Reply:
Not making this change (or the other one right below) as there are usages in CodeAction.cs that are outside the scope of this PR.